docs/LOGGING.md

```diff
@@ -1,6 +1,6 @@
 # EVE LOGGING PIPELINE
 
-This document describes the different stages logs on a EVE edge node go through before being delivered to cloud. It also describes the various congestion points in the current log pipeline along with the places where logs could leak (dropped on the floor) with the current state of EVE code.
+This document describes the different stages logs on a EVE edge node go through before being delivered to cloud. It also describes the various congestion points in the current log pipeline along with the places where logs could leak (dropped on the floor) with a small chance.
```

Comment:
I don't understand what the text with your change is supposed to be telling me.

Reply:
@eriknordmark i can add more explicit to that: when first device bootup before 'newlogd' starts, if the device goes down the log will be lost; and if the CPU is busy, and 'newlogd' is not fast enough to drain the queue from memlogd before saving into log files.

Reply:
But those are not "congestion points" which this sentence is about. I think having those details in a some section further down the document which covers the difference reasons for loosing log entries, and how that can be observed (counters?) would make more sense than putting the details here. Thus the sentence could be "It also describes when and where log entries can be lost."

Reply:
ok